### PR TITLE
Accounts Revamp: Wallet Edit-Config

### PR DESCRIPTION
**Pull request by rauljordan**
**What type of PR is this?**

> Feature

**What does this PR do? Why is it needed?**

This PR includes the ability to edit wallet configurations for Prysm by running `prysm.sh validator wallet-v2 edit-config`. It is only used at the moment to edit remote wallet configuration options, such as gRPC TLS credentials and a gRPC address.

It adds the ability for a keymanager configuration to implement the `Stringer` interface, allowing for pretty-printing when doing fmt.Printf.

**Which issues(s) does this PR fix?**

Part of #6220

### CODE REVIEW

File: validator/accounts/v2/wallet.go [hunk 1]
```diff
@@ -249,11 +249,8 @@ func (w *Wallet) WriteFileForAccount(ctx context.Context, accountName string, fi
 // and writes it to the wallet path.
 func (w *Wallet) WriteKeymanagerConfigToDisk(ctx context.Context, encoded []byte) error {
 	configFilePath := path.Join(w.accountsPath, keymanagerConfigFileName)
-	if fileExists(configFilePath) {
```
Comment (rauljordan) [inline validator/accounts/v2/wallet.go:252]: We should be allowed to overwrite the keymanager config

File: validator/accounts/v2/wallet_create.go [hunk 2]
```diff
@@ -148,14 +168,16 @@ func inputRemoteKeymanagerConfig(cliCtx *cli.Context) (*remote.Config, error) {
...
-			CACertPath:     caCrtPath,
+			ClientCertPath: strings.TrimRight(clientCrtPath, "\r\n"),
+			ClientKeyPath:  strings.TrimRight(clientKeyPath, "\r\n"),
+			CACertPath:     strings.TrimRight(caCrtPath, "\r\n"),
 		},
-		RemoteAddr: remoteAddr,
-	}, nil
+		RemoteAddr: strings.TrimRight(remoteAddr, "\r\n"),
+	}
+	fmt.Printf("%s\n", newCfg)
```
Comment (0xKiwi) [inline validator/accounts/v2/wallet_create.go:179]: Can this log be prettier?

Comment (0xKiwi) [reply validator/accounts/v2/wallet_create.go:179]: Oh, is this using the string function below? That's neat. Can you add it to the PR desc?

Comment (rauljordan) [reply validator/accounts/v2/wallet_create.go:179]: Yep! it prints out nicely given newCfg implements the stringer interface

**Pull request merged**